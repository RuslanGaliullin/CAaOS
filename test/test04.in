Points: (0.000000, 0.000000); (0.000000, 0.000000); (0.000000, 0.000000); (0.000000, 0.000000)
All points belong to the one circle

Calculation time = 0.928807
