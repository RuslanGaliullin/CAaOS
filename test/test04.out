Built array B:
59547 26552 32979 70108 22142 40904 80770 98662 75994 4516 33855 67744 26903 97734 71246 34143 43774 48545 14807 22036 26365 15669 77445 13097 49587 50279 2688 15862 27608 91362 26319 57502 29350 96354 63094 58388 22803 1876 99886 78194 81974 58512 61105 36480 20849 53683 10693 69546 37589 34626 

Calculation time = 1.7e-05
